### DISCUSSION TITLE
Unable to get data from nested json list of objects.

### DISCUSSION THREAD

**Discussion by ddjain** [dynamoose/dynamoose#1546] [Q&A] [answered]
- We have a json schema like this 

- We are trying to fetch the data from a element in array of object. i.e mobileNumber,countryCode

```
{
 "personId": {
  "S": "123"
 },
 "age": {
  "N": "30"
 },
 "haircut": {
  "M": {
   "cuttingday": {
    "N": "2"
   },
   "month": {
    "N": "9"
   },
   "year": {
    "N": "2022"
   }
  }
 },
 "phoneNumbers": {
  "L": [
   {
    "M": {
     "countryCode": {
      "S": "+91"
     },
     "mobileNumber": {
      "S": "123456789"
     }
    }
   },
   {
    "M": {
     "countryCode": {
      "S": "+1"
     },
     "mobileNumber": {
      "S": "123456"
     }
    }
   }
  ]
 },
 "userName": {
  "S": "Jerry"
 }
}
```

simple json view--
```
{
 " ": "123",
 "age": 30,
 "haircut": {
  "cuttingday": 2,
  "month": 9,
  "year": 2022
 },
 "phoneNumbers": [
  {
   "countryCode": "+91",
   "mobileNumber": "123456789"
  },
  {
   "countryCode": "+1",
   "mobileNumber": "123456"
  }
 ],
 "userName": "Jerry"
}
```

- We are able to write filter expressions to get the data from attributes like- personId,haircut,month
Here is filter expressions to get data from cuttingday
```
    DynamoDBScanExpression scanExpression =
        new DynamoDBScanExpression().withFilterExpression("haircut.cuttingday = :cuttingday")
            .withExpressionAttributeValues(eav);
```

- We are not able to get the data from fields like - countryCode mobileNumber

- Is there any particular filter expressions from there I can get this thanks in adv. for needful suggestios.
- What is the filter expression which we can use??

**Comment by fishcharlie** [comment, marked as answer]
Closing as a duplicate of https://github.com/dynamoose/dynamoose/issues/1589 to keep everything in one place.
